testsuite/testsuite-management-console-openapi/src/test/java/org/wildfly/swarm/management/console/openapi/ManagementConsoleAndOpenAPITest.java

```diff
@@ -0,0 +1,64 @@
+package org.wildfly.swarm.management.console.openapi;
+
+/**
+ * Copyright 2018 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS
+ * IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.net.URI;
+
+import org.apache.commons.io.IOUtils;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+import org.wildfly.swarm.undertow.WARArchive;
+
+/**
+ * @see THORN-2047
+ *
+ * @author <a href="mailto:manovotn@redhat.com">Matej Novotny</a>
+ */
+@RunWith(Arquillian.class)
+public class ManagementConsoleAndOpenAPITest {
+
+    @Drone
+    WebDriver browser;
+
+    @Deployment
+    public static Archive createDeployment() throws Exception {
+        return ShrinkWrap.create(WARArchive.class);
```

Comment:
Empty deployment?

Reply:
Ah yea, well honestly it doesn't really matter what's in the WAR (can be empty) as long as there are two deployments up and one is implicit (e.g. test is still valid).
I'll fix this.